README.md

```diff
@@ -54,12 +54,13 @@ For instructions on installing the root server, see [the page on installing a ne
 CHANGELIST
 ----------
 
-***Version 2.10.8* ** *- TBD*
+***Version 2.11.0* ** *- TBD*
```

Comment:
are you releasing today?  if so update TBD??

Reply:
Planning to release this week. I want to make a demo video first.